exercises/beer-song/test/Tests.hs

```diff
@@ -1,39 +1,25 @@
-import Test.HUnit (Assertion, (@=?), runTestTT, Test(..), Counts(..))
-import System.Exit (ExitCode(..), exitWith)
-import Beer (sing, verse)
-
-exitProperly :: IO Counts -> IO ()
-exitProperly m = do
-  counts <- m
-  exitWith $ if failures counts /= 0 || errors counts /= 0 then ExitFailure 1 else ExitSuccess
+import  Test.Hspec        (Spec, describe, it, shouldBe)
```

Comment:
I just noticed the extra space, do all of the test files have this?

Okay, looked at triangle and it doesn't maybe accidental then?